js/ai/src/generate.ts

```diff
@@ -343,16 +351,11 @@ export async function generate<
     resolvedOptions.output.format = 'json';
   }
   const resolvedFormat = await resolveFormat(registry, resolvedOptions.output);
-  const instructions = resolveInstructions(
-    resolvedFormat,
-    resolvedSchema,
-    resolvedOptions?.output?.instructions
-  );
 
   const params: GenerateActionOptions = {
     model: resolvedModel.modelAction.__action.name,
     docs: resolvedOptions.docs,
-    messages: injectInstructions(messages, instructions),
```

Comment:
@mbleigh do you remember why injection was done here? It's redundant.... Action handles it all.

Reply:
I don't remember, but I'm slightly worried it's load-bearing 😅

Reply:
I've done a lot of testing... this appears to be redundant.